docs/viz.md

```diff
@@ -373,6 +373,7 @@ color_function_type | property to determine type of expression used by Mapbox to
 line_color | property to determine choropleth border line color | '#FFFFFF'
 line_stroke | property to determine choropleth border line stroke (one of solid (-), dashed (--), dotted (:), dash dot (-.)) | 'solid' or '-'
 line_width | property to determine choropleth border line width | 1
+line_opacity | opacity of choropleth line layer | 1.0
```

Comment:
I'd suggest the default is opacity `1`

Reply:
@ryanbaumann Updated the docs to match this, thanks.  It is already 1 in the viz.py.